#goal definition
float32 targ_x
float32 targ_y
float32 targ_z
float32 direction_x
float32 direction_y
float32 direction_z
float32 search_distance
---
#result definition
float32 final_x
float32 final_y
float32 final_z
---
#feedback
float32 current_x
float32 current_y
float32 current_z
